features/features.go

```diff
@@ -168,13 +168,6 @@ var (
 						enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 						mustRegister()
 
-	FeatureGateGCPLabelsTags = newFeatureGate("GCPLabelsTags").
-					reportProblemsToJiraComponent("Installer").
-					contactPerson("bhb").
-					productScope(ocpSpecific).
-					enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
```

Comment:
To promote a feature to GA, it is expected first that you add the default feature set here, and then clean up the gating in a later release

Reply:
@JoelSpeed Thank you reviewing the changes!
I have made the required changes. PTAL.